corehq/apps/registration/templates/registration/partials/start_trial_modal.html

```diff
@@ -19,7 +19,7 @@ <h2 class="text-center">
              id="get-trial-cta-form-content" ></div>
         
         {# Schedule Call Choices #}
-        <div id="choose-callback-options" hidden>
+        <div id="choose-callback-options" style="display: none;">
```

Comment:
I think `class="hide"` was what @orangejenny suggested earlier? were you able to try this or did a problem come up with it by any chance? :)

Reply:
Oh yes!
I thought it was `style="display:none;"`
I will update it in a minute.
Thanks for pointing it out.

Reply:
Hey,
I tried out with `class = hide` class and it did not work as expected. Played around a bit with it, it happens out that implementation of `hide` class is `display: none !important;` and `fadeIn()` cannot overwrite !important. 
I create a [pen](https://codepen.io/amitphulera/pen/poJWMpe) to test it out.
How should I go about it?

Reply:
In modal code before, It was `style="display:none`, that's why I changed all `hidden` attribute to it.

Reply:
How about adding a generic class with a transition on visibility/opacity (I don't know offhand if you need both or just one) to [utils.less](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less)? It would be nice for us to have a fade class that plays nicely with `.hide`.

@biyeun lmk if you disagree.

Reply:
I like this idea! :)

Reply:
> How about adding a generic class with a transition on visibility/opacity (I don't know offhand if you need both or just one) to [utils.less](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/less/_hq/utils.less)? It would be nice for us to have a fade class that plays nicely with `.hide`.

Hey @orangejenny,

I created a simple functionality that fulfills the purpose in [Codepen](https://codepen.io/amitphulera/pen/poJWMpe)
 I have created an animation class `fadeIn` class which the element will have and we will toggle `hide` class with JQuery.

Should I implement it here?

And I got to know that [`.hide` has been deprecated](https://getbootstrap.com/docs/3.3/css/#helper-classes-show-hide)  from Bootstrap 3 and they recommend using .hidden class. 

Should we also use `.hidden` instead of `.hide`?

Reply:
Noted. I will make the changes to PR.

Reply:
Done and deployed to staging.